tests/php81_readonly.phpt

```diff
@@ -19,6 +19,7 @@ echo ast_dump($node), "\n";
 $node = ast\parse_code($code, $version=85);
 echo ast_dump($node), "\n";
 --EXPECTF--
+    Deprecated: ast\parse_code(): Version 60 is deprecated in %sphp81_readonly.php on line 12
```

Comment:
Weird bit of indentation here.

Reply:
Strange, I probably had that left over when deleting lines and reading in the `.out` file. Removed